src/azure-cli/azure/cli/command_modules/container/_params.py

```diff
@@ -79,6 +79,13 @@ def load_arguments(self, _):
 
     with self.argument_context('container create', arg_group='Confidential Container Group') as c:
         c.argument('cce_policy', help="The CCE policy for the confidential container group")
+        c.argument('allow_privilege_escalation', options_list=['--allow-escalation'], help="Allows whether a process can gain more privileges than its parent process.", action='store_true')
+        c.argument('privileged', help='The flag to determine if the contianer permissions is elevated to Privileged', action='store_true')
+        c.argument('run_as_user', help="Sets the User GID for the container")
+        c.argument('run_as_group', help="Sets the User UID for the container")
```

Comment:
Please use the first-person voice in the help message
```suggestion
        c.argument('allow_privilege_escalation', options_list=['--allow-escalation'], help="Allow whether a process can gain more privileges than its parent process.", action='store_true')
        c.argument('privileged', help='The flag to determine if the contianer permissions is elevated to Privileged', action='store_true')
        c.argument('run_as_user', help="Set the User GID for the container")
        c.argument('run_as_group', help="Set the User UID for the container")
```